core-web/libs/sdk/client/src/lib/editor/listeners/listeners.ts

```diff
@@ -80,6 +80,11 @@ export function listenEditorMessages() {
                 break;
             }
         }
+
+        if (event.data.name === NOTIFY_CUSTOMER.EMA_SCROLL_INSIDE_IFRAME) {
+            const scrollY = event.data.direction === 'up' ? -120 : 120;
```

Comment:
We might be able to release with this BUT this number can be more or less based on how close of the edge the drag event is.